### PR TITLE
Strip keypair authentication

### PR DESCRIPTION
**Pull request by shal**

### CODE REVIEW

**Comment by yivo** [comment]
@ashanaakh You are missing a lot of things. Websocket API must be refactored to use JWT. You should not rewrite existing code (example: auth middleware). Also don't forget to cleanup helpers everywhere. Be VERY attentive.

**Comment by shal** [comment]
@yivo, It is not finished, I just want you to check if I do it in right way. 

Sorry, what do you mean

> should not rewrite existing code (example: auth middleware).

Should I completely remove `keypairs` and from peatio?

Old auth miidleware was:
```ruby
class Middleware < Grape::Middleware::Base
  def before
    if auth_by_keypair?
      auth = KeypairAuthenticator.new(request, params)
      env['api_v2.keypair_token']       = auth.authenticate!
      env['api_v2.authentic_member_id'] = env['api_v2.keypair_token'].member_id
    elsif auth_by_jwt?
      env['api_v2.authentic_member_email'] = \
        JWTAuthenticator.new(headers['Authorization']).authenticate!
    end
  end
end
```

File: app/api/api_v2/websocket_protocol.rb [hunk 1]
```diff
@@ -42,14 +39,16 @@ def handle(message)
     private
 
     def send(method, data)
-      payload = JSON.dump({method => data})
+      payload = JSON.dump(method: data )
```
Comment (yivo) [inline app/api/api_v2/websocket_protocol.rb]: @ashanaakh Fix please.

**Review by yivo — CHANGES_REQUESTED** [review]

**Comment by shal** [comment]
@yivo Fixed

**Comment by ashanaakh** [comment]
@yivo Fixed

**Review by yivo — APPROVED** [review]

**Pull request merged**